docs/themes/site/site.yaml

```diff
@@ -9,5 +9,5 @@ streams:
 google_analytics_code: UA-57144786-2
 github:
     position: top                                   # top | bottom | off
-    tree: https://github.com/select2/docs/blob/develop/
-    commits: https://github.com/select2/docs/commits/develop/
+    tree: https://github.com/select2/select2/blob/develop/
```

Comment:
Should this include `docs/` at the end since the docs are now located in a subdirectory instead of the root?